website/docs/cloud-docs/vcs/troubleshooting.mdx

```diff
@@ -101,6 +101,18 @@ Terraform Cloud does not allow you to change the API URL for an existing VCS con
 1. [Delete each affected module](/cloud-docs/api-docs/private-registry/modules#delete-a-module), then [create a new module](/cloud-docs/api-docs/private-registry/modules#publish-a-private-module-from-a-vcs) from the new connection's version of the relevant repo.
 1. Delete the old VCS connection.
 
+
+### Reauthorizing VCS OAuth Providers
+
+In the event a VCS OAuth connection breaks, you can reauthorize an existing VCS provider while retaining any vcs connected resources. i.e (Workspaces)
+
+To utilize this feature, do the following:
+- Navigate to the VCS Provider page within your organization settings and you should see an "Reauthorize" button on the existing VCS provider. 
+- After pressing the Reauthorize button, you will receive a confirmation dialog to confirm the action.
+- Once confirmed, you will be redirected to your VCS Provider and be prompted for your to authorization.
+
+-> **Note:** We recommend only using this feature to fix broken vcs connections. Additionally we recommend using the same VCS authorized account as when the connection was first created, to avoid changes in permissioning to your repositories. 
```

Comment:
When we say "permission changes" do we mean like --> you could accidentally give someone access to a repo you don't want them to have? How would that work exactly?

Also what do you think about this edit? I wanted to be very specific about the instructions so I went through the majority of this workflow in TFC. I made a few edits to adhere to our style guide (active voice, making these into numbered steps because they're a process, and bolding UI elements). Thank you! :) 

```suggestion
If a VCS OAuth connection breaks, you can reauthorize an existing VCS provider while retaining any VCS connected resources, like workspaces. We recommend only using this feature to fix broken VCS connections. We also recommend reauthorizing using the same VCS account to avoid permission changes to your repositories. 

To reauthorize a VCS connection, complete the following steps:
1. Go to your organization's settings and click **Providers** under **Version Control**.
1. Click **Reauthorize** in the **OAuth Token ID** column. 
1. Confirm the reauthorization. Terraform Cloud redirects you to your VCS Provider where you can provide authorization.
```

Reply:
Thanks @laurapacilio for the assist! 

> When we say "permission changes" do we mean like --> you could accidentally give someone access to a repo you don't want them to have? How would that work exactly?

More of the opposite. If you change the account on the VCS end there is a chance the new user doesn't have access to a repository that the former user did so any workspaces, registry modules, policy sets would then be broken. 

I really like the edit. Do you think it would be appropriate to show any image's in this section?